lang/en/cpdlogbook.php

```diff
@@ -22,5 +22,10 @@
 $string['name'] = 'Name';
 $string['points'] = 'Points';
 $string['entryname'] = 'Name';
+$string['entryhours'] = 'Hours';
+$string['entrypoints'] = 'Points';
```

Comment:
I'd just reuse the same strings for name / points above and skip the 'entry' prefix